config/environments/development.rb

```diff
@@ -64,6 +64,8 @@
   # Enable debug mode in development
   config.assets.debug = true
 
+  config.assets.compile = true
```

Comment:
What are the implications of this for making changes to assets in development? Do we have to restart the server every time? Should we make this a conditional for if we have "work on pdf redaction tickets flag turned on"?

Reply:
@littleforest We don't need this anymore. It was not the issue for missing assets